implementations/nestjs-code-first/docker-compose.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+  products:
+    # must be relative to the root of the project
+    build: implementations/nestjs
```

Comment:
should this be `implementations/nestjs-code-first`?

Reply:
It should - fixed! I've been verifying this work outside of Docker so everything will still pass after adjusting this.